package.json

```diff
@@ -27,7 +27,9 @@
     "image"
   ],
   "dependencies": {
-    "electron-dl": "^1.2.0"
+    "electron": "^0.4.1",
```

Comment:
This is something completely different than `electron`. Remove. Electron internally adds the `electron` builtin. I've already fixed the issue in master. Just rebase.